security-force-monitor/sfm-cms
# Add association and emplacement tenure models

## Overview

It's possible for an organization to be based in an area or site for multiple ranges of dates. Our current data model does not provide for this to happen. This PR adds an intermediate model, `Tenure`, for both types of relationship, that associates a relationship with optional start and end dates. 

Connects #787

### Notes

There were a number of missing migrations generated when I added the new models, so I've included them in this PR.

## Testing Instructions

* View the example page provided in Slack and confirm it renders without error. Confirm areas of operation are listed in reverse chronological order. Also confirm there is one site.
* Visit a few other organization and person pages and confirm they load without error.
